src/main/java/org/carlmontrobotics/robotcode2023/commands/SetArmWristPosition.java

```diff
@@ -17,6 +16,7 @@ public class SetArmWristPosition extends ConditionalCommand {
     private final static double armSideTransferOffsetFromVertical = Units.degreesToRadians(45);
 
     public SetArmWristPosition(double armPos, double wristPos, Arm arm) {
+        // TODO: Alex simplify this command
```

Comment:
TODO

Reply:
is simple this way
only will fix if have time

Reply:
This is actually going to need a fair amount of work but let's leave that until after the merge.